plugins/kubernetes/test/models/kubernetes/role_test.rb

```diff
@@ -332,22 +342,14 @@ def commit
       role.defaults[:replicas].must_equal 1
     end
 
-    it "does not fail without spec" do
```

Comment:
I think this is still a valid case, but maybe rewrite as `it "does not fail without primary kind" do`